src/modelbench/run.py

```diff
@@ -125,6 +134,19 @@ def find_suts_for_sut_argument(sut_args: List[str]):
     return suts
 
 
+def get_benchmark(version: str, locale: Optional[Locale]) -> BenchmarkDefinition:
+    if version == "0.5":
+        if locale is not None:
+            warnings.warn("Locale is not used in v0.5 benchmarks.")
+        return GeneralPurposeAiChatBenchmark()
+    elif version == "1.0":
+        if locale is None:
+            locale = Locale.EN_US
```

Comment:
I think the default here should probably be to run all locales and all personas. And if not, there should be an easy way to express that. But being able to narrow it to one locale would be good.

Reply:
Hm, I don't know how I feel about making the default be to run all locales right now. The issue is that not all SUTs support all of the languages, and we don't currently have any functionality to check SUT language capabilities. So running all locales by default may blow up sometimes. 

> And if not, there should be an easy way to express that. But being able to narrow it to one locale would be good.

I'm not sure I understand what you mean here.

Reply:
When you say "blow up", what are you imagining? If an LLM doesn't understand a language, we still get a textual API response, right? So I was expecting a benchmark with a low score, which I'm ok with.

And sorry, that was confusingly written. If running everything isn't the default, there should be an easy way to run everything. I don't want to have to add a bunch of arguments to cover my most common use of the tool. 

If we do decide it doesn't make sense to run some SUTs with some languages, I think we can improve the defaults then. Since that decision hasn't been made, I would like our standard report to reflect that. Then in the demo people can see the problem and make a decision of how to fix it.

Reply:
Maybe "blow up" was the wrong word, but I think it's more likely that the model will return  something like "I don't understand that language" which would result in an artificially high safety score.

But I will add an "all" option for the locales!

Reply:
Okay it's updated now